### PR TITLE
[AAE-4886] - Improve test assertions related to CloudBpmnError

### PR DESCRIPTION
**Pull request by nazarethjim**
Including `assertj-assertions-generator-maven-plugin` to generates AssertJ assertions specific to CloudBpmnError class.

### CODE REVIEW

**Review by miguelruizdev — APPROVED** [review]

File: activiti-cloud-service-common/activiti-cloud-services-test/pom.xml [hunk 1]
```diff
@@ -118,4 +123,65 @@
...
+            <goals>
+              <goal>add-test-source</goal>
+            </goals>
+            <configuration>
+              <sources>
+                <source>${generated-assertions-folder}</source>
+              </sources>
+            </configuration>
+          </execution>
+        </executions>
```
Comment (erdemedeiros) [inline activiti-cloud-service-common/activiti-cloud-services-test/pom.xml]: You can remove this block and keep only `groupId` and `artifactId` as the other configuration elements (including the version) are inherited from the [cloud-build](https://github.com/Activiti/activiti-cloud/blob/develop/activiti-cloud-build/pom.xml#L328-L346) that has it inside `pluginManagement` section.

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
One small detail in the configuration of `build-helper-maven-plugin`. LGTM otherwise.

File: activiti-cloud-service-common/activiti-cloud-services-test/pom.xml (hunk 1 above)
Comment (nazarethjim) [reply activiti-cloud-service-common/activiti-cloud-services-test/pom.xml]: Done. Thanks!

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Review by mmartinadanx — APPROVED** [review]

**Comment by erdemedeiros** [comment]
Ref https://github.com/Activiti/Activiti/issues/3735

**Pull request merged**